vendor/redux/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "redux"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+
+[dependencies]
+enum_dispatch = "0.3.7"
+fuzzcheck = { git = "https://github.com/openmina/fuzzcheck-rs.git", optional = true }
+linkme = { version = "0.3.22", optional = true }
+paste = "1.0.14"
+serde = { version = "1.0", features = ["derive"], optional = true }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+wasm-timer = { git = "https://github.com/openmina/wasm-timer" }
```

Comment:
Should we be vendoring this too?

Reply:
See https://github.com/o1-labs/mina-rust/issues/1873